google/cloud/firestore_admin_v1/services/firestore_admin/async_client.py

```diff
@@ -361,9 +361,9 @@ async def get_index(
                 maximum=60.0,
                 multiplier=1.3,
                 predicate=retries.if_exception_type(
-                    exceptions.DeadlineExceeded,
                     exceptions.InternalServerError,
                     exceptions.ServiceUnavailable,
+                    exceptions.DeadlineExceeded,
```

Comment:
The cry goes up both far and near
For ~~Underdog!~~  `gapic-generator-python 0.33.7`